.github/workflows/main.yml

```diff
@@ -63,19 +63,19 @@ jobs:
       if: always()
       run: |
         docker-compose -f docker-compose.yml -f docker-compose.test-e2e.yml ps
-        docker-compose -f docker-compose.yml -f docker-compose.test-e2e.yml logs
+        grep 'desec/' /var/log/syslog
```

Comment:
because `docker-compose.test-e2e.yml` does not change log location for all services, and `docker-compose.yml` defines it to be syslog. (On the contrary, `docker-compose.dev.yml` defines log location to be `"json-file"`.) I'll change the remainder of the services having `"json-file"` in `docker-compose.test-e2e.yml` also to log to syslog, so everything is in one place.